microsoft-edge/webview2/concepts/browserfeatures.md

```diff
@@ -0,0 +1,187 @@
+
+*   `chrome-search://local-ntp/local-ntp.html`  
+*   `edge://application-guard-internals`  
+*   `edge://apps`  
+*   `edge://compat`  
+*   `edge://extensions`  
+*   `edge://favorites`  
+*   `edge://help`  
+*   `edge://management`  
+*   `edge://network-error`  
+*   `edge://new-tab-page`  
+*   `edge://newtab`  
+*   `edge://omnibox`  
+*   `edge://settings`  
+*   `edge://supervised-user-internals`  
+*   `edge://version`  
+
+### Additional shortcuts information  
+
+Key bindings are subject to change when the underlying browser updates.  An accelerator key that is turned off by default may turn on if the feature is supported in WebView2.  However, it's guaranteed that if you set `AreBrowserAcceleratorKeysEnabled` to `FALSE`, all keys that access browser-like features are turned off, and all basic text-editing and movement shortcuts remain turned on.  
+
+The following shortcuts are always turned off in WebView2.
+
+An asterisk \(`*`\) character indicates the shortcut is technically not turned off, but the feature it accesses is turned off or doesn't apply to WebView.  
+
+| Action | Windows |  
+|:--- |:--- |  
+| Add to Favorites | `Ctrl`+`D` |  
+| Add All Tabs to Favorites | `Ctrl`+`Shift`+`D` |  
+| Focus Location | `Ctrl`+`L, Alt`+`D` |  
+| Paste and Go | `Ctrl`+`Shift`+`L` |  
+| Open File | `Ctrl`+`O` |  
+| Read Aloud `*` | `Ctrl`+`Shift`+`U` |  
+| Web Capture `*` | `Ctrl`+`Shift`+`S` |  
+| Sidebar `*` | `Ctrl`+`Shift`+`E` |  
+| Save Page | `Ctrl`+`S` |  
+| Select Last Tab | `Ctrl`+`9` |  
+| Select Next Tab | `Ctrl`+`Tab` |  
+| Select Previous Tab | `Ctrl`+`Shift`+`Tab` |  
+| Select Tab \(1 - 8\) | `Ctrl`+`(1-8)` |  
+| Show Favorites Bar `*` | `Ctrl`+`Shift`+`B` |  
+| Help | `F1` |  
+| Focus Next Pane `*` | `F6` |  
+| Focus Previous Pane `*` | `Shift`+`F6` |  
+| Caret Browsing `*` | `F7` |  
+| Reading View `*` | `F9` |  
+| Focus Menu Bar | `F10` |  
+| Show Identity Menu `*` | `Ctrl`+`Shift`+`M` |  
+| Browser Task Manager `*` | `Shift`+`Escape` |  
+| Edge Feedback `*` | `Shift`+`Alt`+`I` |  
+| Mute Tab `*` | `Ctrl`+`M` |  
+| New Incognito Window | `Ctrl`+`Shift`+`N` |  
+| New Tab | `Ctrl`+`T` |  
+| New Window | `Ctrl`+`N` |  
+| Restore Last Closed Tab | `Ctrl`+`Shift`+`T` |  
+| Focus Favorites | `Alt`+`Shift`+`B` |  
+| Focus Inactive Popup | `Alt`+`Shift`+`A` |  
+| Focus Search | `Ctrl`+`E`, `Ctrl`+`K`, `Search Key` |  
+| Duplicate Tab | `Ctrl`+`Shift`+`K` |  
+| Focus Toolbar `*` | `Alt`+`Shift`+`T` |  
+| Home | `Alt`+`Home`, `Browser Home Key` |  
+| Show App Menu | `Alt`+`E, Alt`+`F` |  
+| Show Favorites | `Ctrl`+`Shift`+`O` |  
+| Show Downloads | `Ctrl`+`J` |  
+| Show History | `Ctrl`+`H` |  
+| Show Reading Mode Bar `*` | `Shift`+`Alt`+`R` |  
+| Show Collections `*` | `Ctrl`+`Shift`+`Y` |  
+
+The following shortcuts are always turned off, except in a pop-up window.  Windows that display when the `NewWindowRequested` event isn't handled.
```

Comment:
The second sentence isn't a complete sentence. Seems like you're trying to define what we mean by pop-up window but don't actually say that. Could remove the term pop-up window like the following?

```suggestion
The following shortcuts are always turned off, except in windows that display when the `NewWindowRequested` event isn't handled.
```

Reply:
Updated.